uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/model/GridColumn.java

```diff
@@ -220,4 +220,34 @@ default CellSelectionStrategy getSelectionStrategy() {
             return HeaderSingleCellSelectionStrategy.INSTANCE;
         }
     }
+
+    /**
+     * Get column width mode
+     * @return
+     */
+    ColumnWidthMode getColumnWidthMode();
+
+    /**
+     * Set column width mode
+     * @return
+     */
+    void setColumnWidthMode(ColumnWidthMode columnWidthMode);
+
+    /**
+     * Enum that identify the width mode of a column
+     */
+    enum ColumnWidthMode {
```

Comment:
It's convention to have Enum members capitalised: https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html

Reply:
Done